### ISSUE TITLE
Update the social media icons and their spacing

### ISSUE THREAD

**Issue by katrinasperry** [codeforpdx/website#29]
Blocked by #28 

Create PR that merges to nav_bar-redesign branch

![image](https://user-images.githubusercontent.com/29134018/142133322-655c307d-1f5d-45a1-8b4f-ce82447d95cf.png)

The navbar html can be found here: `_includes/header.html`

**Comment by kgottfri** [comment]
It looks like the screenshot of the icons in this issue is from the figma mobile nav bar redesign file.  The desktop figma mockup includes the LinkedIn icon and a different github icon.  I wanted to be sure which design we were attempting to work off of for now.

Edit: I noticed the issue was updated to remove the screenshot including the LinkedIn icon and replace with the current screenshot  of just github, meetup, and twitter.  Just want to verify that this is the intended design.
